node/src/chain_specs/manta.rs

```diff
@@ -221,4 +222,203 @@ pub fn manta_testnet_config() -> Result<MantaChainSpec, String> {
 /// Returns the Manta mainnet chainspec.
 pub fn manta_config() -> Result<MantaChainSpec, String> {
     MantaChainSpec::from_json_bytes(&include_bytes!("../../../genesis/manta-genesis.json")[..])
+    // Ok(mainnet::manta_mainnet_config())
+}
+
+pub mod mainnet {
+    use super::*;
+    use hex_literal::hex;
+    use sc_network::config::MultiaddrWithPeerId;
+    use sc_telemetry::TelemetryEndpoints;
+    use sp_core::crypto::UncheckedInto;
+    #[derive(Clone)]
+    struct Collator {
+        acc: AccountId,
+        nodeid: MultiaddrWithPeerId,
+        keys: SessionKeys,
+    }
+    impl Collator {
+        fn new(acc: AccountId, nodeid: MultiaddrWithPeerId, keys: SessionKeys) -> Collator {
+            Self { acc, nodeid, keys }
+        }
+    }
+
+    fn manta_mainnet_genesis(genesis_collators: Vec<Collator>) -> GenesisConfig {
+        const TOTAL_ISSUANCE: Balance = 1_000_000_000 * MANTA;
+        const INITIAL_CONTROLLER_BALANCE: Balance = 5_000 * MANTA;
+        const INITIAL_COLLATOR_BALANCE: Balance = 100 * MANTA;
+        let root_key: AccountId =
+            hex!("fb1818aca757d6bc9aa65c3a94f5a39ff1edbd92e47311c91ace601676168703").into(); // MULTISIG // ss58: 16gEBH3Nq4mtWGg9oFBGveSSoUDcKRABwAq9KESiWKEwb1vQ
+        let endowments = vec![
+            (
+                hex!("4a5d27062c9d0b5747566139c683c17a0e6da8c440c41dfa01725ba664af6977").into(),
+                INITIAL_CONTROLLER_BALANCE,
+            ), // CONTROLLER 1 ss58 12gWDW1z7T1AN1Vm86RcvRdSmyjtJ1Mv4LHmYvmq9LJG8czh
+            (
+                hex!("a4c2839433ae05aa48e615e5a6917cc98e4a2886272f2519347f2be7c0c6853d").into(),
+                INITIAL_CONTROLLER_BALANCE,
+            ), // CONTROLLER 2 ss58 14j2dssnCyQD2xWezTvZmbZPSKDBrwKcAcBtXrE6XgeGfzGx
+            (
+                hex!("3e31eaf98e0f6dd9b088c095757fac00dc9e3fe6af29e42caea94f7d76dd3444").into(),
+                INITIAL_CONTROLLER_BALANCE,
+            ), // CONTROLLER 3 ss58 12QYoFyrrEomSq4tMPxVxnWvPwVVKQHUpDWZuDxcg5Gv3c9E
+            (
+                hex!("0619e4018531059b677d70b476b72431d3e88fe71d2c27bba5d945975e824956").into(),
+                INITIAL_CONTROLLER_BALANCE,
+            ), // COLLATOR 1
```

Comment:
should be CONTROLLER4. and bellow also need to rename the right order

Reply:
good point, the name INITIAL_CONTROLLER_BALANCE is misleading here.
We just decided to give one collator more starting balance for emergency recovery. It is not an actual multisig contoller account